src/index.ts

```diff
@@ -101,6 +101,7 @@ export function apply(ctx: Context, config: Config) {
     .option('noise', '-n <noise:number>', { hidden: restricted })
     .option('strength', '-N <strength:number>', { hidden: restricted })
     .option('undesired', '-u <undesired>')
+    .option('disable-translate', '-d', { hidden: !!ctx.translator && config.translator })
```

Comment:
```suggestion
    .option('noTranslator', '-T', { hidden: () => !ctx.translator || !config.translator })
```

1. 这里逻辑是不是反了？
2. 参数会自动 hyphenate 所以不用担心
3. 使用函数可以支持 hot reload

Reply:
I see. good for it